cabal-install/Distribution/Client/CmdInstall.hs

```diff
@@ -149,6 +155,11 @@ newInstallOptions _ =
     "Set the environment file that may be modified."
     ninstEnvironmentPath (\pf flags -> flags { ninstEnvironmentPath = pf })
     (reqArg "ENV" (succeedReadE Flag) flagToList)
+  -- TODO choose a name. --force-overwrite, --overwrite-symlink, --overwrite...
+  , option [] ["force-overwrite"]
```

Comment:
Is `--force-overwrite` ok?

Reply:
maybe we could go with a more generic 

    --overwrite-policy[=always,never]

which has the benefit of 

1. being extendable to e.g.

        --overwrite-policy[=always,always_warn,ask,never]`

and 

2. allowing for changing what the default should be (e.g. via `~/.cabal/config`) -- which seems to be a good idea given we had some controversy about in the issue discussion

Reply:
1. good idea, doing that.
0. What about scripts? It's another flag that has to be overridden to ensure a consistent behavior.

Reply:
Didn't do the 2nd.
I'll open another issue for that.